modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,346 @@
+// Module included in the following assemblies:
+//CNF-1483 (4.8)
+// * scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+
+[id="reducing-nic-queues-using-the-performance-addon-operator_{context}"]
+= Reducing NIC queues using the Performance Addon Operator
+
+The Performance Addon Operator allows you to adjust the Network Interface Card  (NIC) queue count for each network device by configuring the performance profile. Device network queues allow packets to be distributed among different physical queues and each queue gets a separate thread for packet processing.
+
+In real-time or low latency systems all the unnecessary interrupt request lines (IRQs) that are pinned to the isolated CPUs must be moved to reserved or housekeeping CPUs.
+
+In deployments with applications that require system, {product-title} networking or in mixed deployments with Data Plane Development Kit (DPDK) workloads, multiple queues are needed to achieve good throughput and the number of NIC queues should either be adapted or remain unchanged. For example, to achieve low latency the number of NIC queues for DPDK based workloads should be reduced to just the number of reserved or housekeeping CPUs.
+
+Too many queues are created by default for each CPU and these do not fit into the interrupt tables for house keeping CPUs when tuning for low latency. Reducing the number of queues makes proper tuning possible. Smaller number of queues means smaller number of interrups which then fit in the IRQ table.
+
+[id="adjusting-nic-queues-with-the-performance-profile_{context}"]
+== Adjusting the NIC queues with the performance profile
+
+The performance profile lets you adjust the queue count for each network device.
+
+Supported network devices are:
+
+* Non virtual network devices
+
+* Network devices that support multiple queues (channels)
+
+Unsupported network devices are:
+
+* Pure software network interfaces
+
+* Block devices
+
+* Intel DPDK virtual functions
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift CLI (`oc`).
+
+
+.Procedure
+
+. Log in to the OpenShift Container Platform cluster running the Performance Addon Operator as a user with `cluster-admin` privileges.
+
+. Create a performance profile that is appropriate for your hardware and topology. For more information see the Tuning nodes for low latency with the performance profile section. Alternatively, edit an existing performance profile using the following command:
```

Comment:
Suggestion:
I would recommend adding a link to  Tuning nodes for low latency with the performance profile link in your assembly under .Additional resources. This way the customer a link directly to the location that you are asking them to go to. If you place the .Additional resources right under the include statement it will show up at the end of the module in your assembly.